src/main/java/org/jabref/preferences/AiPreferences.java

```diff
@@ -0,0 +1,40 @@
+package org.jabref.preferences;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+public class AiPreferences {
```

Comment:
Reads like that this could be a `record`.

Reply:
May be.

But other preferences classes are all `class`.
"Code symmetry" from Java by Comparison 😅